src/internal_events/common.rs

```diff
@@ -76,11 +77,9 @@ pub struct StreamClosedError {
 impl InternalEvent for StreamClosedError {
     fn emit(self) {
         error!(
-            message = "Events dropped.",
-            reason = "Downstream is closed.",
+            message = "Failed to forward event(s), downstream is closed.",
             error_code = STREAM_CLOSED,
             error_type = error_type::WRITER_FAILED,
-            intentional = "false",
```

Comment:
Why are dropping this and the `reason`?

Reply:
The motivation for this change was to have a `ComponentEventsDropped` event which handles the requirements for "EventsDropped" (from the instrumentation spec). 
So this error! log becomes just for the "Error" requirements (`component_errors_total`).

Reply:
(With the goal of re-using `ComponentEventsDropped` in other components.